tests/test_profiler/test_comm_prof.py

```diff
@@ -4,11 +4,11 @@
 import torch.distributed as dist
 import colossalai
 from colossalai.utils import free_port, get_current_device
-from colossalai.utils.profiler import enable_communication_prof, communication_prof_show
+from colossalai.utils.profiler import CommProfiler, ProfilerContext
 
 BATCH_SIZE = 1024
 D_MODEL = 1024
-CONFIG = dict(parallel=dict(tensor=dict(mode='1d', size=4)))
+CONFIG = dict()
 
 
 def run_test(rank, world_size, port):

```

Comment:
This is not a valid test. You only make sure the profiler does not fail.

Reply:
It has been deleted.